### ISSUE TITLE
Still undesignated Orf1a:K261N Delta sublineage is dominant in Russia and widespread in Europe

### ISSUE THREAD

**Issue by FedeGueli** [cov-lineages/pango-designation#320]
Orf1a:K261N spotted in various issues here ( Ex.: #268 )
is the dominant lineage in Russia according to COVSpectrum:

In Russia the  share of "non AY.s" B.1.617.2 is 47,60% in the last three months and 82,1% in the last week.

Of these 733 sequences 90,18% has Orf1a:K261N

https://cov-spectrum.ethz.ch/explore/Russia/AllSamples/Past3M/variants?pangoLineage=B.1.617.2

And it is widely spread in several european countries:
![Schermata 2021-11-14 alle 08 03 44](https://user-images.githubusercontent.com/87669813/141671048-f14b4613-8cbe-446c-8d91-09483b98d5db.png)

Country	Total Variant Sequences	
Belgium	988	
Bulgaria	126	
Canada	253
Czechia	506	
Denmark	5234	
Estonia	857	
France	1834	
Germany	4506	
Iceland	44	
Ireland	303	
Israel	255	
Italy	998	
Japan	51	
Lithuania	175	
Luxembourg	193	
Montenegro	26	
Netherlands	1219	
Norway	776	            
Poland	851	
Russia	661	
Slovakia	283	
Sweden	3598	
Switzerland	375	
United Kingdom	2410	
United States	6403	
Croatia	166	
Ecuador	75	
Georgia	48	
Greece	291	
Guadeloupe	9	
India	34	
Spain	153
Turkey	571	
Slovenia	439	
South Korea	160	
and so on.

I think it is very urgent to designate it to understand well the actual wave in Europe for citizens and scientists.

And keep in mind that ~14% of AY.43 sequences in europe are missassigned carrying instead the Orf1a:K261N mutation distinctive of this lineage
https://cov-spectrum.ethz.ch/explore/Europe/AllSamples/Past3M/variants?pangoLineage=AY.43

@chrisruis @AngieHinrichs @corneliusroemer

**Comment by FedeGueli** [comment]
![chrome_screenshot_1636877615576](https://user-images.githubusercontent.com/87669813/141673177-1211b4dc-e956-489f-a648-ecc78b2d8be7.png)

It is a self evident clade.

**Comment by MCB6** [comment]
> Of these 733 sequences 90,18% has Orf1a:K261N

we discussed it at length in the Orf7a:P45L thread, only I used the GISAID's standard NSP2:K81N designation for it. Basically Orf1a:K261N is called in almost all Russian samples, but (older) Orf7a:P45L - only in some of them. But it isn't a reversion mutation back to Proline, as @corneliusroemer showed. Rather, it is a possible genotyping problem combined with a flaky phylogenetic tree building, so some the supposedly Orf7a:45P genomes are scattered across many branches.

I am just using NSP2_K81N to query Russian uploads on GISAID now. Out of most recently collected 100 samples, 20% are classified as AY.43 and most of the rest as B.1.617.2. 

My gut feeling is that epidemiologists in Eastern Europe don't realize that there is a strong strain continuity since spring or summer, because Pango labels keep on shifting, and that it may help them if they realize that they are dealing with the same strain for a second wave in a row, rather than with a changed variety of the virus.

**Comment by MCB6** [comment]
PS: adding graphs. Orf1a:K261N aka NSP2:K81N has been dominant in Russia since June, flat at 10-25% rate in a number of East and Central European countries, but interestingly, it's in retreat in Finland even as they experience a fall wave
![russian-delta](https://user-images.githubusercontent.com/93294184/141697023-f4379a6e-1981-4748-b063-da14a60f8eef.png)
![russian-delta1](https://user-images.githubusercontent.com/93294184/141697024-3c356bc5-196a-45f3-a1d8-c2332c1e2d0b.png)
https://cov-spectrum.ethz.ch/explore/Russia/AllSamples/from=2021-06-09&to=2021-11-14/variants?aaMutations=Orf1a%3AK261N&pangoLineage=B.1.617.*

**Comment by FedeGueli** [comment]
Thx @MCB6 for your analysis.

There are a lot of sublineages of K261N , but it is becoming hard to follow them without a designation.
Plus missassignements create rumour and we dont need that, Europe is hardly hit right now and i dont see how to keep this clade undesignated.

Missassignments of it arent just in AY.43 so it creates an unnecessary mess:
![Uploading chrome_screenshot_1636965706562.png…]()

**Comment by MCB6** [comment]
Since Israel experienced such a wild swing from mostly Orf7a:P45L to mostly negative genotypes, I decided to check USHER trees for various timepoints of the Israeli Delta wave. I wondered if I will clearly see two distinct branches.
The June collection dates yield the following (one minor but apparently separate branch is visible)
![image](https://user-images.githubusercontent.com/93294184/142135388-52100737-3fb5-4c8e-84a5-08532c37ea0e.png)
Restricting to Orf7a:P45L only, I see only the "lower sub-branch" of the main branch
![image](https://user-images.githubusercontent.com/93294184/142135577-abc6d616-1e03-4776-920f-69102eae24db.png)
And looking at the October genomes, I see the same branches as in June, just shifted to the right a little, of course
![image](https://user-images.githubusercontent.com/93294184/142135842-47e43697-cc81-4998-84da-7f2e87f66e76.png)

So it looks like most samples with or without Orf7a:P45L are closely related (although there may exist a minor completely unlinked branch without it). Formally these two types of samples mostly map to adjacent sub-branches, but I can't tell if this separation is a tree-building artifact. But at the very least they look closely related.

**Comment by chrisruis** [comment]
Thanks @FedeGueli We've added this as AY.122 in [v1.2.95](https://github.com/cov-lineages/pango-designation/releases/tag/v1.2.95) to start on the branch with G1048T (Orf1ab:K261N). From what I can see, it looks like Orf7a:P45L drops in and out across the clade so this doesn't look like an ideal marker for a sublineage so for now we've designated the complete clade as AY.122 with the exception of the clade that was AY.12.

Lineage AY.12 clusters within the AY.122 clade. We've therefore renamed AY.12 to AY.122.1 and withdrawn the AY.12 label. AY.122.1 is downstream of C23934T (S:T791I).

**Comment by FedeGueli** [comment]
Thank you vey much @chrisruis , good choice on AY.12 'renaming'

Just to say all the credits about this lineage go to @c19850727 🙏

There are a few sublimeages already proposed for AY.122. 

i'll share the issues further below so you could monitor them.easily.
